tests/mongo-go-driver.yml

```diff
@@ -36,6 +36,11 @@ results:
 
       # error terminating open transactions: (CommandNotFound) no such command: 'killAllSessions'
       - go.mongodb.org/mongo-driver/mongo/integration/unified/TestUnifiedSpec
+      
+      # https://github.com/FerretDB/FerretDB/issues/3601
```

Comment:
That issue is unrelated to that apparent regression

Reply:
Ok, I will revert the changes and investigate the issue separately.